src/redux/books/books.js

```diff
@@ -1,5 +1,11 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+// import store from "../configureStore";
```

Comment:
### OPTIONAL
 - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉